README.md

```diff
@@ -17,7 +17,16 @@ To preview your changes locally:
 ```
 cargo install mdbook --version "^0.4" --force
 mdbook build --open
-mdbook test
+```
+
+`mdbook test` does not currently work, see: https://github.com/rust-lang/mdBook/issues/394
```

Comment:
I suggest linking our issue (the one you opened) instead of the mdbook one

Reply:
changed